readme.md

```diff
@@ -144,6 +144,11 @@ New node operators may also use the following peer addresses:
 16d9c23623e42723dfcf3dcbb11d98d989689a7a@35.166.247.98:28006
 ```
 
+## Localnet with docker
```

Comment:
Id prefer we dont put this in the main readme. However totally happy to start a doc under `./doc/` for how to run the intrgration test suite?